File: Codeforcess_Plm_Java_C/BoyOrGirl.java
import java.util.*;

public class BoyOrGirl {
    public static void main(String[] args) {
       
        Scanner input = new Scanner(System.in);
        String username = input.next();

        Set<Character> uniqueLetters = new HashSet<>();

        
        for (int i = 0; i < username.length(); i++) 
        
        {
            uniqueLetters.add(username.charAt(i));
        }

       
        if (uniqueLetters.size() % 2 == 0) 
        
        {
            System.out.println("CHAT WITH HER!");
        } 
        
        else
        
        {
            System.out.println("IGNORE HIM!");
        }
    }
}

File: Codeforcess_Plm_Java_C/Using C/boyOrgirl.c
#include <stdio.h>
#include <string.h>

int main() {
    char username[105];   
    int freq[256] = {0}; 
    int uniqueCount = 0;

    scanf("%s", username);

    int len = strlen(username);

    for (int i = 0; i < len; i++) {
        unsigned char c = username[i];
        if (!freq[c]) {   
            freq[c] = 1;
            uniqueCount++;
        }
    }

    if (uniqueCount % 2 == 0) {
        printf("CHAT WITH HER!\n");
    } else {
        printf("IGNORE HIM!\n");
    }

    return 0;
}
